daemons/attrd/attrd_attributes.c

```diff
@@ -142,7 +142,16 @@ attrd_add_value_xml(xmlNode *parent, const attribute_t *a,
     crm_xml_add(xml, PCMK__XA_ATTR_SET_TYPE, a->set_type);
     crm_xml_add(xml, PCMK__XA_ATTR_SET, a->set_id);
     crm_xml_add(xml, PCMK__XA_ATTR_USER, a->user);
-    pcmk__xe_add_node(xml, v->nodename, v->nodeid);
```

Comment:
There's only one remaining caller of `pcmk__xe_add_node()` that doesn't pass 0 for `nodeid`. We could just get rid of it.

Reply:
I'll leave that for someone else :)